src/main/java/frc/robot/commands/ShootingPosCommands/PodiumPosition.java

```diff
@@ -21,11 +22,12 @@ public void execute(){
         pivot.pivotPodiumPosition();
         elevator.podiumPosition();
         flywheel.podiumShot();
+        isAtPodium = true;
     }
 
     @Override
     public boolean isFinished(){
-        return true;
+        return isAtPodium;
```

Comment:
Same comment here as above, this will finish immediately

Reply:
got it :)